### PR TITLE
Pd 1290 update admin user content for 24.10

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR updates GettingStarted/Install, Configure, and Migrate articles mentioning the admin user that changes to truenas_admin in systems clean installing 24.10. Systems upgrading with the admin user retain that user name.
Updates tutorial articles and snippets containing content on how to install the administrator account in systems with only root user accounts.
Updates UI ref user screen content and all screenshots with changes made to the UI Add/Edit User screen.
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/GettingStarted/Configure/FirstTimeLogin.md [hunk 1]
```diff
@@ -103,8 +100,8 @@ If using the CORE **Update** UI option, log into SCALE with the CORE root user c
 {{< include file="/static/includes/CreateAdminDisableRoot.md" >}}
 {{< /expand >}}
 
-### Logging In as Admin
-After setting up the admin user from one of the scenarios documented above, enter **admin** and the password to log into SCALE.
+### Logging In as Truenas_Admin
```
Comment (Mrt134) [inline content/SCALE/GettingStarted/Configure/FirstTimeLogin.md:103]: Could we make this more generic to be more consistent with some of the other generic section headers and avoid having to update later if/when the default name changes again?

```suggestion
### Logging In as the Default Administrator
```

File: content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md [hunk 2]
```diff
@@ -82,7 +82,7 @@ If this process fails, retry using the iso file method above.
    ![SCALESidegradeReboot](/images/SCALE/SystemSettings/SidegradeRestart.png  "Reboot to Finish")
 
 After TrueNAS SCALE reboots, sign in with the root user credentials used in CORE.
-Uploading the CORE config file deletes the admin user account created during a clean install and therefore requires you to [recreate it](#recreating-the-admin-user-account).
+Uploading the CORE config file deletes the **truenas_admn** user account created during a clean install and therefore requires you to [recreate it](#recreating-the-admin-user-account).
```
Comment (Mrt134) [inline content/SCALE/GettingStarted/Migrate/MigratingFromCORE.md:85]: typo
```suggestion
Uploading the CORE config file deletes the **truenas_admin** user account created during a clean install and therefore requires you to [recreate it](#recreating-the-admin-user-account).
```

File: static/includes/AddAdminUserAccount.md [hunk 3]
```diff
@@ -1,36 +1,37 @@
...
-All CORE systems migrating to SCALE, and all Angelfish and early Bluefin releases of SCALE upgrading to 22.12.3+ or to later SCALE major versions should create and begin using an admin user instead of the root user. 
-After migrating or upgrading from CORE or a pre-SCALE 22.12.3 release to a later SCALE release, use this procedure to create the Local Administrator user.
+All CORE systems migrating to SCALE, and all early SCALE releases such as Angelfish and pre-22.12.3 Bluefin, that migrate or upgrade to later SCALE major versions must create and begin using an admin user instead of the root user.
+After migrating or upgrading from CORE or a pre-SCALE 22.12.3 releases to a later SCALE release, use this procedure to create the administrator user.
 
 Go to **Credentials > Local Users** and click **Add**.
 
-Enter the name to use for the administrator account. For example, *admin*.
-You can create multiple admin users with any name and assign each different administration privileges.
+Enter the name to use for the administrator account. For example, **truenas_admin**, **admin**, etc.
```
Comment (Mrt134) [inline static/includes/AddAdminUserAccount.md:8]:
```suggestion
Enter the name to use for the administrator account. For example, **truenas_admin**, **tnas-admin**, etc.
As a security recommendation, try to create unique account names that aren't easy to guess.
```

**Review by Mrt134 — CHANGES_REQUESTED** [review]
Looks good - found a couple things to fix

File: content/SCALE/GettingStarted/Configure/FirstTimeLogin.md (hunk 1 above)
Comment (micjohnson777) [reply content/SCALE/GettingStarted/Configure/FirstTimeLogin.md:103]: The default admin is truenas_admin, and I wanted to be explicit about this as we are used to just admin.

File: static/includes/AddAdminUserAccount.md (hunk 3 above)
Comment (micjohnson777) [reply static/includes/AddAdminUserAccount.md:8]: they cannot enter tnas_admin, they have to enter truenas_admin as that is the default user created... unless eng has changed this in the past week. the two possible system-created administrators are truenas_admin or admin based on how you get to 24.10.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**